modules/api/src/main/java/org/apache/fluo/api/config/SimpleConfiguration.java

```diff
@@ -242,6 +243,27 @@ public void setProperty(String key, String value) {
     internalConfig.setProperty(key, value);
   }
 
+  public void setProperties(String key, String... values) {
+    if (values == null) {
```

Comment:
Thinking it would be best to throw an exception if null.

Reply:
`values` can be null, but you can also have an array with some of the elements null. Both cases should be an `IllegalArgumentException`, I think.